_posts/2015-06-20-mental-number-line.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Mass muddles mental number line
+author: neuroamanda
+---
+
+_Judgment of number influenced by other magnitudes and senses_
+
+People appear to have an inherent bias when it comes to number.
+They tend to count from left to right and are quicker to react in psychological tests when smaller numbers are on the left and larger numbers are on the right, at least in Western cultures; [chicks also display this same behavior](http://www.aaas.org/news/humans-chicks-count-left-right).
+Studies have found this mental number line to be pervasive across tasks where people judge size, duration or emotional expression (preferring frownies on the left and smileys on the right, for example).
+But is the mental number line just about number or is it part of a more general system that extends to any judgment of magnitude?
+
+Researchers sought to test this using a standard experiment with the addition of an irrelevant magnitude, a weight worn on the wrist.
+Twenty-four participants had to quickly and accurately judge whether a number (0-9) on a screen is odd or even.
+They gave their responses using a keyboard, and the experiment was balanced so that the 'odd' and 'even' responses were assigned to keys on the left ('Q') or right ('P') side of the keyboard in half the trials, and reversed in the other half.
```

Comment:
Better! :) Still missing the word "respectively" perhaps, but maybe I'm just being pedantic?